CHANGES.md

```diff
@@ -1,3 +1,18 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured.
+  * The `--proxy-auto-inject` flag  from the `install` command has been completely revamped
+  * The `inject` command proxy options are now converted into config annotations. The annotations ensure that these configs are persisted in subsequent resource updates
+  * Always show open TCP connections in the CLI
+  **Breaking Change** Removed the `--disable-external-profiles` flag from the `install` command. External profiles are now disabled by default, and can be enabled with the new `--enable-external-profiles` flag.
+* Controller
+  * The webhook config now reacts to changes in pods instead of deployments
```

Comment:
I would add a bit more detail, like "; this means we can now auto-inject pods parented not only to Deployments, but also to any kind of workload like Daemonsets and Statefulsets"